src/pemfile.rs

```diff
@@ -1,7 +1,13 @@
-use alloc::{borrow::ToOwned, format, string::String, vec::Vec};
+use alloc::{borrow::ToOwned, vec::Vec};
```

Comment:
I think we'd prefer to see updated imports put in the style the Rustls repo recommends:

```
use alloc::borrow::ToOwned;
use alloc::vec::Vec;
```